packages/x-data-grid/src/tests/quickFiltering.DataGrid.test.tsx

```diff
@@ -16,7 +16,7 @@ import { getColumnValues, sleep } from 'test/utils/helperFn';
 const isJSDOM = /jsdom/.test(window.navigator.userAgent);
 
 describe('<DataGrid /> - Quick filter', () => {
-  const { render, clock } = createRenderer();
+  const { render, clock } = createRenderer({ clock: 'fake' });
```

Comment:
I have changed to have fake timers for the whole file because almost every test in this file used fake timers (except 2 at the bottom).